components/wordpress_org/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "@pipedream/wordpress_org",
+  "version": "1.0.0",
```

According to (lerna.json)(https://github.com/PipedreamHQ/pipedream/blob/master/lerna.json#L7)
```suggestion
  "version": "0.3.4",
```